elasticpypi/config.py

```diff
@@ -1,7 +1,26 @@
-import json
 import os
 
+WATCHED=[
+    'SERVICE',
+    'STAGE',
+    'BUCKET',
+    'TABLE',
+    'USERNAME',
+    'PASSWORD',
+    'OVERWRITE'
+]
+
+TRUTHY = ['TRUE', 'true', 1]
+FALSEY = ['FALSE', 'false', 0]
```

Comment:
These are largely untested. I am going to remove the `overwrite` option soon as it unused.